forem/forem
# Updates ESLint rules to error on default imports

## What type of PR is this? (check all applicable)


## Description
Updates our eslint rules to error on default JS exports. This provides consistency when importing javascript across the app. 

## Related Tickets & Documents
See #12383 

## QA Instructions, Screenshots, Recordings
I did some manual sanity testing, but it would be great if someone could just quickly hop on this branch and check to make sure nothing is broken! It most likely isn't though. 


## Added tests?

- [x] No, and this is why: This PR updates a lint rule and the accompanying violations.

## Added to documentation?

      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/)

![dancing janitor](https://media.giphy.com/media/xThuWcUB3BESQVfNpm/giphy.gif)
